### PR TITLE
wazuh-puppet tests on GitHub Actions

### LINKED ISSUE #263: Migrate tests to Github Actions
Hi all,

Creating required Workflows in Github's actions would ease the testing process of the new Pull Requests opened. It would also make easier the task of publishing the Wazuh module in forge.puppet.com

### Tasks

- [x] Investigate about Github Actions and create a simple run to test it

- [x] Evaluate if it passes all the requirements: OS variety, multi containers, performance, enough hardware specs, etc...

- [x] Create a workflow to test new PRs

Best regards,

Jose

### PR DESCRIPTION
**Pull request by Zenidd**
Hi team,

This PR enables the kitchen tests on this repo through GitHub actions. It breaks the current dependency on Jenkins.
This PR closes #263 

Greetings,
JP

### CODE REVIEW

File: .github/actions/test_manager_and_agent/entrypoint.sh [hunk 1]
```diff
@@ -0,0 +1,55 @@
+#! /usr/bin/env bash
+set -e
+
+echo `systemctl status`
```
Comment (xr09) [inline .github/actions/test_manager_and_agent/entrypoint.sh:4]: Why `echo command` instead of just `command`?

File: .github/workflows/main.yml [hunk 2]
```diff
@@ -0,0 +1,41 @@
+
+name: Kitchen tests
+
+on: [push]
```
Comment (xr09) [inline .github/workflows/main.yml:4]: Will this be triggered on every PR?

File: .github/actions/test_manager_and_agent/entrypoint.sh (hunk 1 above)
Comment (Zenidd) [reply .github/actions/test_manager_and_agent/entrypoint.sh]: These commands were placed with debug purposes on the initial commits. I'm removing them completely.  Done on 
11104d4

File: .github/workflows/main.yml (hunk 2 above)
Comment (Zenidd) [reply .github/workflows/main.yml:4]: With the current setting, it will be triggered on any push on every PR.

**Pull request merged**